### PR TITLE
nightMode: Add option to follow media query 'prefers-color-scheme: dark'

### PR DESCRIPTION
**Pull request by larsjohnsen**
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

Tested in browser: Firefox 71, Chrome 79

### CODE REVIEW

File: lib/modules/nightMode.js [hunk 1]
```diff
@@ -137,7 +140,9 @@ module.beforeLoad = () => {
 	toggle.addCLI('ns');
 	if (module.options.nightSwitch.value) toggle.addMenuItem(i18n('nightModeToggleTitle'), 7, '☽', '☀');
 
-	handleAutomaticNightMode();
+	if (module.options.automaticNightMode.value !== 'none') {
```
Comment (andytuba) [inline lib/modules/nightMode.js]: How about `MediaQueryList.addListener("that dark theme css", handleAutomaticNightmode)`

https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/addListener

**Comment by andytuba** [comment]
Nice addition?

**Comment by BenMcGarry** [comment]
@larsjohnsen is this good to merge?

**Comment by larsjohnsen** [comment]
Should be good.

**Comment by BenMcGarry** [comment]
Tested on Edge Version 80.0.361.69 (Official build) (64-bit). All good.

**Pull request merged**